.github/workflows/focus-ios-smoke.yml

```diff
@@ -55,15 +54,16 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
+                xcodebuild_test_plan: ['SmokeTest']
+                # xcodebuild_test_plan: ['SmokeTest', 'FullFunctionalTests']
```

Comment:
To turn on full functional tests, I can just update the matrix.

Reply:
Not sure if we would like to have one github action for smoke and another for full functional... what do you think?

Reply:
I'm ok either way.